test/common/CMakeLists.txt

```diff
@@ -22,18 +22,26 @@ add_library(${_COMMON_TARGET_NAME} INTERFACE)
 target_include_directories(${_COMMON_TARGET_NAME} INTERFACE ${_COMMON_INCLUDE_DIRECTORY})
 
 include(${_COMMON_COMPILE_OPTIONS_FILE})
+message(STATUS "ALPAKA_DEV_COMPILE_OPTIONS: ${ALPAKA_DEV_COMPILE_OPTIONS}")
```

Comment:
Do we still need this?

Reply:
I would like to keep this for debugging purposes. It only writes the additional compiler options we use for the tests.